Api/Services/UserService.cs

```diff
@@ -188,65 +188,61 @@ public async Task<bool> IsUniqueLoginAsync(string login)
     /// <returns></returns>
     public async Task<List<UserEmployeeVM>> GetEmployeesAsync(string userId)
     {
-        return await dbContext.Users
+        var users = await dbContext.Users
             .Where(u => u.EmployerId == userId)
-            .Select(u => new UserEmployeeVM
+            .Include(u => u.Employments)
+            .ThenInclude(e => e.Restaurant)
+            .Select(u => new
             {
-                UserId = u.Id,
-                Login = u.UserName!,
-                FirstName = u.FirstName,
-                LastName = u.LastName,
-                BirthDate = u.BirthDate!.Value,
-                PhoneNumber = u.PhoneNumber!,
-                Employments = u.Employments
+                u.Id,
+                u.UserName,
+                u.FirstName,
+                u.LastName,
+                u.BirthDate,
+                u.PhoneNumber,
+                u.Employments,
+                u.PhotoFileName
+            })
+            .ToListAsync();
+
+        var employeeVMs = new List<UserEmployeeVM>();
+
+        foreach (var user in users)
+        {
+            var employeeVM = new UserEmployeeVM
+            {
+                UserId = user.Id,
+                Login = user.UserName!,
+                FirstName = user.FirstName,
+                LastName = user.LastName,
+                BirthDate = user.BirthDate!.Value,
+                PhoneNumber = user.PhoneNumber!,
+                Employments = user.Employments
                     .Where(e => e.DateUntil == null)
                     .Select(e => new EmploymentVM
-                    {   
+                    {
                         EmploymentId = e.Id,
                         RestaurantId = e.RestaurantId,
                         IsBackdoorEmployee = e.IsBackdoorEmployee,
                         IsHallEmployee = e.IsHallEmployee,
-                        RestaurantName = e.Restaurant.Name,
+                        // Check for null Restaurant to avoid NullReferenceException
+                        RestaurantName = e.Restaurant != null ? e.Restaurant.Name : "Unknown",
```

Comment:
Asocjacja z restaurant jest wymagana, pole Restaurant będzie null tylko jeśli zapomniemy dodać Include 
```suggestion
                        RestaurantName = e.Restaurant.Name,
```